app/src/main/java/com/github/fribourgsdp/radio/data/Playlist.kt

```diff
@@ -120,6 +153,20 @@ data class Playlist (override var name: String, var genre: Genre) : Nameable {
         return db.registerPlaylist(this)
     }
 
+    /**
+     * @return true if all songs have lyrics
+     */
+    fun allSongsHaveLyrics(): Boolean{
+        return songs.all { it.lyrics != "" }
+    }
+
+    /**
+     * @return true if no song has lyrics
+     */
+    fun noSongHaveLyrics(): Boolean{
+        return songs.all { it.lyrics == "" || it.lyrics == "---"}
+    }
+
```

Comment:
Could you please use the two constants defined in MusixmatchLyricsGetter ?

Reply:
Yes of course, in any cases, I was planning on using the method of song that returns whether it has lyrics or not that you told me you would implement.

Reply:
Oh I see the one you mean. We will se which PR will be merged first but we should look at this carefully when we will merge